src/socket/src/sink.rs

```diff
@@ -210,15 +210,15 @@ impl ExclusiveFlvSink {
     }
 
     pub fn id(&self) -> ConnectionFd {
-        self.fd
+        self.fd.clone()
     }
 }
 
 impl Clone for ExclusiveFlvSink {
     fn clone(&self) -> Self {
         Self {
             inner: self.inner.clone(),
-            fd: self.fd,
+            fd: self.fd.clone(),
```

Comment:
Clippy doesn't like this but it's needed for wasm as the `fd` is a `String` (doesn't implement `Copy`) in wasm32.